### PR TITLE
[New Rule] Azure Automation Account Created

### LINKED ISSUE #107: [New Rule] Azure Automation Account Created
## Description

Azure Automation accounts can be used to automate management tasks and orchestrate actions across systems. An adversary may create an Automation account in order to maintain persistence in their target's environment.

```
event.module:azure and event.dataset:azure.activitylogs and event.category:Administrative and azure.activitylogs.operation_name:MICROSOFT.AUTOMATION/AUTOMATIONACCOUNTS/WRITE and event.outcome:Success
```

## Required Info

- **Eventing Sources:**

`filebeat-*`

- **Target Operating Systems:**

- **Platforms**

Azure

- **Target ECS Version:** `1.5.0`
- **New fields required in ECS for this?** No
- **Related issues or PRs** None

## Optional Info
- **References:**
  - https://powerzure.readthedocs.io/en/latest/Functions/operational.html#create-backdoor
  - https://github.com/hausec/PowerZure
  - https://posts.specterops.io/attacking-azure-azure-ad-and-introducing-powerzure-ca70b330511a
  - https://azure.microsoft.com/en-in/blog/azure-automation-runbook-management/

## Example Data

image

### PR DESCRIPTION
**Pull request by threat-punter**
## Issues

Resolves #107 

## Summary
Identifies when an Azure Automation account is created. Azure Automation accounts can be used to automate management tasks and orchestrate actions across systems. An adversary may create an Automation account in order to maintain persistence in their target's environment.

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: rules/azure/persistence_azure_automation_account_created.toml [hunk 1]
```diff
@@ -0,0 +1,63 @@
+[metadata]
+creation_date = "2020/08/18"
+ecs_version = ["1.5.0"]
+maturity = "development"
```
Comment (threat-punter) [inline rules/azure/persistence_azure_automation_account_created.toml:4]:
```suggestion
maturity = "production"
```

**Comment by Samirbous** [comment]
Looks neat, no specific observation.

**Review by Samirbous — APPROVED** [review]
Looks neat!

File: rules/azure/persistence_azure_automation_account_created.toml [hunk 2]
```diff
@@ -0,0 +1,62 @@
+[metadata]
+creation_date = "2020/08/18"
+ecs_version = ["1.5.0"]
```
Comment (peasead) [inline rules/azure/persistence_azure_automation_account_created.toml:3]:
```suggestion
ecs_version = ["1.6.0"]
```

**Pull request merged**